index.bs

```diff
@@ -33,6 +33,10 @@ urlPrefix: https://w3c.github.io/sensors; spec: GENERIC-SENSOR
     text: default sensor
     text: construct a sensor object; url: construct-sensor-object
     text: sensor type
+urlPrefix: https://w3c.github.io/screen-orientation; spec: SCREEN-ORIENTATION
+  type: dfn
+    text: current orientation type
+    text: dom-screen; url: dom-screen
```

Comment:
I think, if text and link are the same, you can leave just 'text:' part

Reply:
Usually that works, but in this case `text: current orientation type` creates a link to https://w3c.github.io/screen-orientation/#current-orientation-type that does not exist. Should go to https://w3c.github.io/screen-orientation/#dfn-current-orientation-type thus explicit `text: current orientation type; url: dfn-current-orientation-type` needed. ReSpec bug perhaps.